packages/fast-colors/src/range.ts

```diff
@@ -1,4 +1,4 @@
-import * as Chroma from "chroma-js";
+import { default as Chroma } from "chroma-js";
```

Comment:
if this is the fix, can you just do `import Chroma from "chroma-js"`?

Reply:
Confirmed and updated for all packages.